### PR TITLE
Set up next base styling

### PR DESCRIPTION
**Pull request by danielireson**
Adds base styling using CSS variables, taking inspiration from [Bulma](https://bulma.io) base styles.

### CODE REVIEW

File: styles/variables.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+:root {
```
Comment (Jeanice-N) [inline styles/variables.css:1]: Out of curiousity, where are these values coming from? Did you take them from design tokens?

Comment (Jeanice-N) [reply styles/variables.css:1]: Nevermind, guess you're taking them bulma

**Comment by Jeanice-N** [comment]
LGTM. Great work on the refactoring too! 👌

**Review by Jeanice-N — APPROVED** [review]

File: styles/variables.css (hunk 1 above)
Comment (danielireson) [reply styles/variables.css:1]: From both actually, the colours are from design tokens and the base styles are primarily ported from [bulma sass files](https://github.com/jgthms/bulma/tree/master/sass).

**Comment by danielireson** [comment]
Thanks for reviewing, I'll merge these then and let's see how we get on with them.

**Pull request merged**